src/redux/state.js

```diff
@@ -1,82 +1,128 @@
-import { rerenderEntireTree } from '../render';
-
-const state = {
-  profilePage: {
-    posts: [
-      {
-        id: '1',
-        message: 'Hi, how are you',
-        likesCount: '12',
-      },
-      {
-        id: '2',
-        message: 'It`s my first posts',
-        likesCount: '7',
-      },
-    ],
-    newPostText: '',
+const ADD_POST = 'ADD-POST';
+const UPDATE_NEW_POST_TEXT = 'UPDATE-NEW-POST-TEXT';
```

Comment:
Consider moving these variables to file named "actionTypes.js"

Reply:
/fxd